tox.ini

```diff
@@ -1,7 +1,7 @@
 [tox]
 envlist =
        py27-{flake8,docs},
-       {py27,py33,py34}-django{1.6,1.7,1.8}-drf{2.4,3.0,3.1}
+       {py27,py33,py34,py35}-django{1.6,1.7,1.8,1.9,1.10}-drf{2.4,3.0,3.1,3.3,3.5}
```

Comment:
I'm leaving them in for now, as we are not breaking compatibility for now. It's on the ToDo list though for later.